Makefile

```diff
@@ -109,8 +111,8 @@ p4a_android_distro: needs-android-dirs check-android-clean
 # Update the python-for-android project bootstrap, discarding any changes that are made to committed files
 # this should be the usually run command in normal workflows.
 .PHONY: p4a_android_project
-p4a_android_project: install-tar p4a_android_distro create-strings needs-version
-	$(P4A) bootstrap $(ARCH_OPTIONS) --version=$(APK_VERSION) --numeric-version=$(BUILD_NUMBER)
+p4a_android_project: install-tar p4a_android_distro create-strings
+	$(P4A) bootstrap $(ARCH_OPTIONS) --version="None" --numeric-version=1
```

Comment:
Yep, that's precisely it. P4A bakes this information into the build.gradle file during its bootstrapping of the build process, but I wanted to decouple from P4A as much as possible - mostly to make the Java project files stable, rather than continually changing based on P4A's builds (also it changes the build.gradle depending on whether it's a dev or production build, which didn't work well with committing these files to the repo).